app/views/layouts/application.html.haml

```diff
@@ -33,7 +33,7 @@
     :javascript
       document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');
     %a.govuk-skip-link{href: "#main-content"} Skip to main content
-    %header.govuk-header{"data-module": "header", role: "banner"}
```

Comment:
Super minor, it's useful to link to the changelog entry in the commit message where possible.

Reply:
will do an interactive rebase and amend the commit to link to the relevant changelog entry

Reply:
linked to the changelog entry in https://github.com/DFE-Digital/teacher-vacancy-service/pull/1057/commits/50079f5af39a77ee141205863631f619408f461f